### ISSUE TITLE
http and https issue

### ISSUE THREAD

**Issue by ArmorDarks** [stephanmax/grunt-responsive-images-extender#16]
Hi @stephanmax!

Following html:

```html
<img src='https://placekitten.com/100/100'>
```

will throw an error:

![chrome_2017-06-22_22-11-55](https://user-images.githubusercontent.com/4460311/27451489-d62c13b2-5797-11e7-996c-a0b56117fe2c.png)

It seems that plugin doesn't ignore remote files.

**Comment by stephanmax** [comment]
Indeed. Thanks a lot for pointing this out. I'll get to it after the weekend!
